docker/docker-compose-igpu.yaml

```
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  wyoming-glados:
+    image: captnspdr/wyoming-glados:latest-igpu
+    container_name: wyoming-glados
+    ports:
+      - 10201:10201
+    environment:
+      - streaming=true
+    restart: unless-stopped
+    runtime: nvidia
+    environment:
```

YAMLlint (1.37.1)

[error] duplication of key "environment" in mapping

(key-duplicates)